int64 id
int64 stimulus_object
int64 haptic_time
